Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -279,8 +279,9 @@ Optional<PaintableBox::ScrollbarData> PaintableBox::compute_scrollbar_data(Scrol
     auto scrollport_size = direction == ScrollDirection::Horizontal ? padding_rect.width() : padding_rect.height();
     if (scroll_overflow_size == 0)
         return {};
-
-    auto thumb_length = scrollport_size * (scrollport_size / scroll_overflow_size);
+    auto min_thumb_length = 50;
```

Comment:
@AtkinsSJ I'm trying to understand how you structuring code in C++. Shall we move it to `ScrollbarData` struct as a constant to make an association?

https://github.com/LadybirdBrowser/ladybird/blob/84ab8bf7973147d054a99445943b9da930ceb0a2/Userland/Libraries/LibWeb/Painting/PaintableBox.h#L224

Something like this:

```cpp
struct ScrollbarData {
    CSSPixelRect thumb_rect;
    CSSPixelFraction scroll_length;

public:
    static CSSPixels constexpr min_thumb_length = CSSPixels::from_raw(50);
};
```

Reply:
Just having a variable here is fine. (Or making it `auto const` if you want.)

If at some point we need this value in multiple places, then it could be moved into a constant like that.

Reply:
Got it, thanks :+1: